tests/types/gem-test.json

```
@@ -0,0 +1,145 @@
+{
+  "$schema": "https://packageurl.org/schemas/purl-test.schema-1.0.json",
+  "tests": [
+    {
+      "description": "Java gem can use a qualifier. Rountrip an input purl to canonical.",
```

```suggestion
Ruby gems can use a qualifier. Rountrip an input purl to canonical.
```
Should this read as `Ruby gem...` or `Ruby gems...`? Other gems can have qualifiers on them
ex. https://rubygems.org/gems/nokogiri/versions

---

This make sense ... the spec is really for any platform https://guides.rubygems.org/specification-reference/#platform=  and we could even add that list from https://github.com/rubygems/rubygems/blob/20b093f58fbc008e634b187263f947d112e8c1af/bundler/lib/bundler/rubygems_ext.rb#L54 in the main spec, or at least point to it.

Let me create an issue for that.

and about:
> Should this read as Ruby gem... or Ruby gems...?

Gems or RubyGems  I guess ... the intent there was to talk Gems for the Java platform.

---

I will address this with:
- https://github.com/package-url/purl-spec/issues/539